### PR TITLE
Limit event types length

### PR DESCRIPTION
**Pull request by Ana06**
I've limit the event types length to be a multiple of 15, adding validation and tests.

I've created a variable `LENGTH_STEP` in `EventType` for if we want to change the step (10 minutes instead of 15 for example). I plan to use this variable in the schedule also.

### CODE REVIEW

**Comment by Ana06** [comment]
There is a test failing in my computer: 

> Failures:
>   1) conference/show.html.haml renders banner component

But it doesn't fail here... :confused:

**Comment by ChrisBr** [comment]
@Ana06 please provide some stacktrace / error message then!

File: spec/models/event_type_spec.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      expect(build(:event_type)).to be_valid
+    end
+
+    it { is_expected.to validate_presence_of(:title) }
+    it { is_expected.to validate_presence_of(:minimum_abstract_length) }
+    it { is_expected.to validate_presence_of(:maximum_abstract_length) }
+
+    describe 'has a valid length' do
+      it 'length is greater than 0' do
+        should validate_numericality_of(:length).
```
Comment (ChrisBr) [inline spec/models/event_type_spec.rb]: You can also split this test!

Comment (ChrisBr) [inline spec/models/event_type_spec.rb]: http://betterspecs.org/#single

**Comment by Ana06** [comment]
I closed the console and when running the tests again I realised that if I run only the file where the error is supposed to be, then there is no error. But If I run all tests, the error appears. It is very strange :confused: Here is the start and end of the error (is is a really long error):

image

image

File: spec/models/event_type_spec.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        is_greater_than(0)
+      end
+
+      it 'length is multiple of LENGTH_STEP' do
```
Comment (ChrisBr) [inline spec/models/event_type_spec.rb]: The test is not testing what it says

Comment (ChrisBr) [inline spec/models/event_type_spec.rb]: I'd group it into some valid and invalid context

File: spec/models/event_type_spec.rb (hunk 1 above)
Comment (ChrisBr) [inline spec/models/event_type_spec.rb]: And please use the new expect syntax
http://betterspecs.org/#expect

File: spec/models/event_type_spec.rb (hunk 2 above)
Comment (ChrisBr) [inline spec/models/event_type_spec.rb]: Also the positive case is missing

Comment (ChrisBr) [inline spec/models/event_type_spec.rb]: Something like this:
```
 describe 'length' do
     it 'is valid when length is multiple of LENGTH_STEP' do
      ...
    end 
     it 'is not valid when length is multiple of LENGTH_STEP' do
      ...
    end
end
```

**Comment by ChrisBr** [comment]
LGTM, @hennevogel what do you think?

**Comment by hennevogel** [comment]
If it looks good for you let's go :-)

**Pull request merged**